src/Swarm/Game/CESK.hs

```diff
@@ -164,7 +166,7 @@ data Frame
     --   if there is one, to the output of the first command).
     FBind (Maybe Var) Term Env
   | -- | Apply specific updates to the world and current robot.
-    FImmediate WorldUpdate RobotUpdate
+    FImmediate Const [WorldUpdate] [RobotUpdate]
```

Comment:
Is the `Const` just for better error messages?  So we know what command was ultimately responsible for generating this `FImmediate` frame?

Reply:
Exactly, `FImmediate` was definitely created by some command which we use in the error message.

I will note this in the docstring. 👍